src/StatusWriter.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+// #include "schemas/fws0_fwr_status_generated.h"
+#include "rapidjson/document.h"
+
+namespace FileWriter {
+namespace Status {
+
+class StreamMasterStatus;
+class StreamerStatusType;
+class StreamerStatisticsType;
```

Comment:
`StreamerStatisticsType` is struct originally, causes warning.

Reply:
:white_check_mark: (these classes have been removed and replaced)